ross/defects/crack.py

```diff
@@ -496,9 +498,9 @@ def _mayes(self, ap):
         return K
 
     def _get_coefs(self, coef):
-        x = scipy.io.loadmat(
-            "/home/izabela/Documents/Projeto EDGE Petro/ross/tools/data/PAPADOPOULOS_c"
-        )
+
+        cwd = os.getcwd()
+        x = scipy.io.loadmat(cwd + "/tools/data/PAPADOPOULOS_c")
```

Comment:
I would recommend using pathlib for this.
It is the library that we are using in most of our code for dealing with paths.

Reply:
Thank you Rafael, I will make the necessary changes.